README.md

```diff
@@ -22,7 +22,8 @@ docker-machine create --driver amazonec2 kvstore
 ```
 eval $(docker-machine env kvstore)
 export KV_IP=$(docker-machine ip kvstore)
-docker run -d -p 8500:8500 -h consul --restart=always progrium/consul -server -bootstrap
+docker-machine ssh kvstore
```

Comment:
Can you make it a Note? Like:

If you want to SSH-in to the physical server then run `docker-machine ssh kvstore` instead of `export KV_IP=$(docker-machine ip kvstore)`

The thing is that `export KV_IP=$(docker-machine ip kvstore)` will export all DOCKER* related environment variables so that the instruction that comes next (docker run) will work.